proxy/logging/LogFile.cc

```diff
@@ -255,15 +255,15 @@ LogFile::close_file()
       ::close(m_fd);
       Debug("log-file", "LogFile %s (fd=%d) is closed", m_name, m_fd);
       m_fd = -1;
+      RecIncrRawStat(log_rsb, this_thread()->mutex->thread_holding, log_stat_log_files_open_stat, -1);
     } else if (m_log) {
       m_log->close_file();
```

Comment:
Might want to add a return value for `BaseLogFile::close_file()`

Reply:
Yeah, that should be straightforward.